docs/dependencies.rst

```diff
@@ -92,3 +92,90 @@ need to use them (the most common base app is used for `Electron applications
 <https://github.com/flathub/io.atom.electron.BaseApp>`_). However, if your
 application uses a large, complex or specialized framework, it is a good
 idea to check for available base apps before you start building.
+
+Extensions
+----------
+
+Runtimes and applications can define extension points which allow optional
+additional runtimes to be mounted at a specified location inside the sandbox
+when they are present on the system. Typical uses for extension points include
+translations for applications, debuginfo for sdks, or adding more functionality
+to the application. Some software refers to these extensions as "Add-ons".
+
+By convention, extension points follow the application ID of the application in
+question, followed by a generic term the extension is conveying. For example,
+OBS Studio supports plugins to extend the application's functionality.
+OBS Studio Flatpak has an extension point that goes by
+``com.obsproject.Studio.Plugin``, as "Plugin" conveys the extension type.
+
+We can use the ``add-extensions`` property in the manifest to specify where
+and how should extensions be installed:
+
+.. code-block:: yaml
+
+  add-extensions:
+    org.flatpak.App.ExampleExtension:
+      directory: my-dir # Installs extensions in $FLATPAK_DEST/my-dir
+      version: '1.0' # Branch version of extension
+      versions: 21.08;22.08beta # Supported extension versions
+      subdirectories: true # Creates $FLATPAK_DEST/my-dir/ExampleExtension directory
```

Comment:
This is not what the `subdirectories` property does. When set to `false`, only one extension with ID exactly matching the extension point is mounted at specified `directory`. When set to `true`, multiple extensions can be mounted under the same `directory`, with the extension point being a "prefix" for the extensions IDs, while the rightmost part of an extension ID is used as a subdirectory name.